client/src/Components/Feature/Feature.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { ReactNode } from 'react';
+import { useFeatureFlags } from '../FeatureFlagsContext/FeatureFlagsContext';
+
+interface FeatureProps {
+  flag: string;
+  children: ReactNode;
+}
+
+const Feature = ({ flag, children }: FeatureProps) => {
+  const { featureFlags } = useFeatureFlags();
+
+  if (!featureFlags) {
+    return <div>Feature flags not loaded yet</div>;
```

Comment:
Would it be better to display a loading spinner rather than this text?

Reply:
Should probably returns the element in case the feture flag is not there. We should probablt assume true.